stats_test.py

```diff
@@ -29,6 +29,6 @@ def test_range_():
 
 def test_standard_deviation():
     """Tests the SD function"""
-    assert stats.standard_deviation(A) == 2.0
-    assert stats.standard_deviation(B) == 7.07107
-    assert stats.standard_deviation(E) == 38.34058
+    assert stats.standard_deviation(A) == 1.63299
```

Comment:
So this is a great way to check if you implemented the refactoring of standard_deviation() correctly. None of these values should have changed from their previous ones. Your refactor should still yield the same results as what calling statistics.stdev() did.